dll/win32/browseui/ACLCustomMRU.cpp

```diff
@@ -33,12 +33,22 @@ STDMETHODIMP CACLCustomMRU::Next(ULONG celt, LPWSTR *rgelt, ULONG *pceltFetched)
     if (INT(m_ielt) >= m_MRUData.GetSize())
         return S_FALSE;
 
-    size_t cb = (m_MRUData[m_ielt].GetLength() + 1) * sizeof(WCHAR);
+    CStringW str = m_MRUData[m_ielt];
+
+    if (!m_bTypedURLs)
+    {
+        // Erase the last "\\1" etc. (indicates SW_* value)
+        INT ich = str.ReverseFind(L'\\');
+        if (ich >= 0)
+            str = str.Left(ich);
+    }
```

Comment:
Suppose that the value is instead: `some_directory\notepad.exe` (instead of, `notepad.exe\1`). How would this situation behave in Windows? Would it work as well with your new modification?

Reply:
I confirmed Windows was smarter than me. I added file existence check. OK.

Reply:
Case 1:
![case1](https://user-images.githubusercontent.com/2107452/95074907-41eda600-074a-11eb-9d07-88c2de9ebc21.png)
Case 2:
![case2](https://user-images.githubusercontent.com/2107452/95074903-40bc7900-074a-11eb-8429-80903a53d084.png)

Reply:
Case 1 contains `"\1"`. That works well.
Case 2 doesn't contain `"\1"`. That is merged with `"C:\Windows"`. (There is no `C:\WinDows`)

Reply:
Windows is case insensitive by default, so `Windows` or `WinDows` is the same for it.

Reply:
Reverted.